src/validation/validation-refinements.ts

```diff
@@ -194,8 +197,8 @@ export const validateBeaconMetadataReferences: SuperRefinement<{
     }
 
     // Find any missing chains - where the chain is specified in pricingCoverage but not in the beacon itself
```

Comment:
Is the opposite being checked? This entire file is unreadable to me.

Reply:
I don't think it is. Bedirhan proposed a good idea which was adding logic to the normalization script to add the chains to pricingCoverage based on the chains listed for each beacon/beaconSet

Reply:
`pricingCoverage` shouldn't exist in the operations repo in the first place, and it's not even clear that we will need these values in the market, so this is not very important